gluster-exporter/conf/conf.go

```diff
@@ -22,13 +23,24 @@ type CollectorConfig struct {
 
 // LoadCollectorsConfig loads the collectors configuration
 func LoadCollectorsConfig(confFilePath string) string {
-	fileObj, err := os.Open(confFilePath)
+	fileObj, err := os.Open(filepath.Clean(confFilePath))
 	if err != nil {
 		msg := fmt.Sprintf("Failed loading config file: %s", confFilePath)
 		return msg
 	}
-	defer fileObj.Close()
-	data, _ := ioutil.ReadAll(fileObj)
+	defer func() {
+		err = fileObj.Close()
+		if err != nil {
+			// TODO: Log error
+			return
```

Comment:
We need an error message here. Anyway after we merge updated .toml based config, this function returns an error object and same can be used.

Reply:
we need log here, error message returned from defer will not reach the caller.